### PR TITLE
Adding instructions on spinning up a Node.js server

### PR DESCRIPTION
**Pull request by pacostas**
Adding info on how to setup and start a Node.js server.

### CODE REVIEW

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (abstractj) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]: I'd suggest removing the entire block above. Cover `git` steps is out of the scope for the Keycloak, plus Node.js developers are already familiar with `.gitignore`.

Comment (pacostas) [reply securing_apps/topics/oidc/nodejs-adapter.adoc]: Yeap!

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (abstractj) [inline securing_apps/topics/oidc/nodejs-adapter.adoc:131]: @pacostas I believe it would make sense to indent the code below. Other than that, changes lgtm

**Review by abstractj — DISMISSED** [review]
LGTM

**Comment by abstractj** [comment]
@jonkoops waiting for your review, before merging it.

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
In that case a Keycloak deployment is necessary to access Keycloak admin console.
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
Please visit links on how to deploy a Keycloak admin console with 
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
    app.listen(3000, function () {
        console.log('App listening on port 3000');
    });
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
Last but not least, let's set up our server to listen for HTTP requests on port 3000 by adding the following code to `main.js`:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
and configure Keycloak middleware in Express, by adding at the code below:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
In the root directory of your project create a file called `server.js` and add the following code:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
Install the `express-session` dependency:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
    npm install express-session
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
To start the `server.js` script, add the following command in the 'scripts' section of the `package.json`:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
Now we have the ability to run our server with following command:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
        secret: 'mySecret',
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
    npm install express
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
In order to do so, first we have to install Express:
```

File: securing_apps/topics/oidc/nodejs-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
then require Express in our project as outlined below:
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by pacostas** [comment]
@jonkoops Thank you for the Review! :)

**Comment by abstractj** [comment]
@pacostas could you please squash your commits into a single one?

**Comment by pacostas** [comment]
@abstractj Squashed

**Review by abstractj — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**
